docs/basics/101-131-summary.rst

```diff
@@ -0,0 +1,28 @@
+.. _summary_python:
+
+Summary
+-------
+
+DataLad's Python API makes all of DataLad's functionality available in
+Python, either as standalone functions that are exposed via ``datalad.api``,
+or as methods of the ``Dataset`` class.
+This provides an alternative to the command line, but it also opens up the
+possibility of performing DataLad commands directly inside of scripts -- a
```

Comment:
The somewhat indicates that only Python can do script. But Matlab users can use datalad as well (via standard system calls) -- just not that convenient.